README.md

```diff
@@ -88,7 +88,7 @@ IngestionPipeline::from_loader(FileLoader::new(".").with_extensions(&["rs"]))
 
 - Extremely fast streaming pipeline with parallel processing
 - Integrations with OpenAI, Redis, Qdrant and Treesitter
-- Bring your own transformers by extending straightforward traits.
+- Bring your transformers by extending straightforward traits.
```

Comment:
```suggestion
- Bring your own transformers by extending straightforward traits.
```

Bring your own is definately intended here as a figure of speech. There's no intention at this point to implement a full feature set like llamaindex or langchain.